src/gitmoji.js

```diff
@@ -168,17 +179,28 @@ class GitmojiCli {
 			},
 			{
 				name: 'reference',
-				message: 'Issue / PR reference #:',
+				message: 'Issue / PR reference:',
 				validate(value) {
 					if (value === '') {
 						return true;
 					}
 					if (value !== null) {
-						const validReference = value.match(/(^[1-9][0-9]*)+$/);
+						let validReference = '';
+						let errorReference = '';
+						switch (config.get('issueFormat')) {
+							case 'jira':
+								validReference = value.match(/^([a-zA-Z]{1,10}-[0-9]+)$/g);
```

Comment:
Never worked with Jira, are you sure that regular expression follows the issue format ? 

As I'm seeing in [Jira Docs](https://confluence.atlassian.com/jirasoftwarecloud/linking-issues-776997756.html) the pattern is like `XXX-11111111` three letters and no limit for the number, is that correct?